docs/pages/indexer.mdx

```diff
@@ -265,3 +271,22 @@ To understand the result, it is easiest to look at it inside Node.
 ## Debugging
 
 [See here](/reference#mud-server-components).
+
+## Indexing other blockchains
+
+To index a different blockchain, you specify these environment variables:
+
+| Variable        | Value                                                                                                    |
+| --------------- | -------------------------------------------------------------------------------------------------------- |
+| RPC_HTTP_URL    | The URL to access the blockchain using HTTP                                                              |
+| RPC_WS_URL      | The URL to access the blockchain using WebSocket (for notifications)                                     |
```

Comment:
```suggestion
| RPC_WS_URL      | The URL to access the blockchain using WebSocket                                     |
```
I'd remove this qualifier, the differences between http and ws are more nuanced but not really relevant here